### DISCUSSION TITLE
为什么界面没有配置存储策略的选项？

### DISCUSSION THREAD

**Discussion by zhucan** [cloudreve/cloudreve#1899] [Q&A]
![image](https://github.com/cloudreve/Cloudreve/assets/13997135/65d6fada-ec59-4fe3-9232-27c5c4320aa4)

**Comment by zk19940207** [comment]
同问

**Reply by zhucan** [reply]
建议你用filestash，国内的东西不太活跃，不好用

**Reply by iwvw** [reply]
进后台配置

**Comment by crwoers** [comment]
![image](https://github.com/cloudreve/Cloudreve/assets/124245254/ba7eeeef-676c-43d5-ba28-e5d95ebd08fd)
![image](https://github.com/cloudreve/Cloudreve/assets/124245254/dbbf18f6-ef98-44ce-9c10-c95b9cb5233b)
